### ISSUE TITLE
Invalid warning on Mixins should come before declarations when having multiple includes

### ISSUE THREAD

**Issue by jjanssen** [sasstools/sass-lint#80]
I am currently working with sass-lint 1.1.0

When using multiple mixins within a selector the rule for 'Mixins should come before declarations' starts throwing invalid warnings when using 3 or more. In another scenario the prepending selector can also cause this issue to throw a warning.

I have setup this little scss file to reproduce this issue:

```
@mixin foo($content) {
    content: $content;
}

@mixin bar {
    background-color: red;
}

@mixin hello {
    border-radius: 3px;
}


.beverages {

    .coffee {
        width: 100px;
        height: 100px;
    }    

    .tea {
        @include foo('yo');
        @include bar;
        @include hello;
        display: block;
        width: 100px;
        height: 100px;
    }
}
```

This snippet throws the following warning through the CLI
```
scss/mixin.scss
  24:9  warning  Mixins should come before declarations  mixins-before-declarations
```

This doesn't occur in the following cases:
1. When disabling the selector `.coffee` the warning is gone.
2. When disabling `@include hello;`. The other includes do not make a difference.

**Comment by DanPurdy** [comment]
Thanks for this report @jjanssen it made tracking the issue a little easier. Hopefully fixed now, will just await review from the others.

**Comment by jjanssen** [comment]
No problem! Seemed you guys put a lot of effort in this package the past few weeks. So if I can help by narrowing down the use case then it makes it easier for everybody :)
